sass/directives/_flyout.scss

```diff
@@ -87,3 +87,117 @@
     }
   }
 }
+
+// Use the mobile-subnav mixin for the subnav bars that appear on our subscription / product pages
+// Displays a single link with a flyout
+// Structure of markup should be:
+@mixin mobile-subnav-flyout() {
+  width: auto; 
+  display: none;
+  position: relative;
+  cursor: pointer;
+
+  @media only screen and (max-width: $small-screen-max) {
+    line-height: 45px;
+    display: block;
+    height: $product_subnav--height;
+    margin-left: -$container-edge-padding;
```

Comment:
Because of the major refactor on the Employer side of things, this was taken out. There is a negative margin in `_product-page.scss`, but its applied to `.product-subnav__left` to make it the full width of the browser in mobile view since its within a padded container.